.coveragerc

```diff
@@ -1,12 +1,11 @@
 [run]
-branch = True
```

Comment:
So this is about three things. The first two are that branch coverage is even slower than regular coverage under PyPy, and that the vast majority of tox.ini I've seen around here don't enable branch coverage. 

They probably don't enable branch coverage because of the third point: with branch coverage on, it's tricky to get and maintain 100% coverage. This project reports 99.33% coverage when branches are on---and those last few lines can be very tricky to get to because they tend to occur in nested conditionals. It's not that the lines aren't being executed, it's that the particular combination of conditionals that got them there only got them there in one way, out of two possibilities.

It just seemed more valuable to me to do what most projects do and be content to have 100% line coverage, rather than struggle to get to 100% branch coverage, or live with only 99% branch coverage.